Tools/autotest/vehicle_test_suite.py

```diff
@@ -4632,11 +4632,17 @@ def install_example_script_context(self, scriptname):
         self.install_example_script(scriptname)
         self.context_get().installed_scripts.append(scriptname)
 
-    def install_test_script_context(self, scriptname):
+    def install_test_script_context(self, scriptnames):
         '''installs an test script which will be removed when the context goes
         away'''
-        self.install_test_script(scriptname)
-        self.context_get().installed_scripts.append(scriptname)
+        if getattr(scriptnames, '__iter__', None) is None:
+            scriptname = [scriptnames]
+        for scriptname in scriptnames:
+            self.install_test_script(scriptname)
+            self.context_get().installed_scripts.append(scriptname)
+
+    def install_test_scripts_context(self, *args, **kwargs):
+        return self.install_test_script_context(*args, **kwargs)
```

Comment:
Why an additional function?

Reply:
> Why an additional function?

Yeah, should have added that comment....

It's just pluralisation to avoid annoying someone who wants to use the function.

Reply:
Why not have the plural function just call the single function repeatedly? Then you don’t have to worry about the type.

Reply:
> Why not have the plural function just call the single function repeatedly? Then you don’t have to worry about the type.

The current implementation allows the user to call either with either parameter.

Trying not to repeat the problem with `set_parameter` vs `set_parameters`, which I'm still mulling up fixing (just to make it more forgiving about calling one instead of the other where getting it exactly right really shouldn't matter).